docs/kernel-policy.md

```diff
@@ -82,19 +48,23 @@ on which Firecracker is run:
 #### Minimal boot requirements
 
 Depending on the source of boot (either from a block device or from an initrd),
-the minimally required guest kernel modules for a successful microVM boot are:
+the minimal configuration for a guest kernel for a successful microVM boot is:
 
 * Booting with initrd:
-  * `CONFIG_BLK_DEV_INITRD`
-    * For aarch64, you also need `CONFIG_VIRTIO_MMIO` (for the serial device).
-    * For x86_64, you also need `CONFIG_KVM_GUEST`.
+  * `CONFIG_BLK_DEV_INITRD=y`
+    * aarch64 `CONFIG_VIRTIO_MMIO=y` (for the serial device).
+    * x86_64 `CONFIG_KVM_GUEST=y`.
 
 * Booting with root block device:
-  * `CONFIG_VIRTIO_BLK`
-  * For x86_64, you also need `CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES` and `CONFIG_KVM_GUEST`.
-
-If you wish to enable boot logs, make sure to also add
-`CONFIG_SERIAL_8250_CONSOLE` and `CONFIG_PRINTK` to the guest kernel config.
+  * aarch64
+    * `CONFIG_VIRTIO_BLK=y`
+  * x86_64
+    * `CONFIG_VIRTIO_BLK=y`
+    * `CONFIG_VIRTIO_MMIO_CMDLINE_DEVICES=y`
+    * `CONFIG_KVM_GUEST=y`.
+
+*Optional*: To enable boot logs set `CONFIG_SERIAL_8250_CONSOLE=y` and
+`CONFIG_PRINTK=y` in the guest kernel config.
 
 ## [Experimental] Snapshot compatibility across kernel versions
```

Comment:
this part could move to `snapshotting/snapshot-support.md`

Reply:
Moved it to `snapshotting/snapshot-support.md`.

Reply:
Looks good. Thank you!